src/main/java/org/candlepin/subscriptions/tally/billing/BillableUsageController.java

```diff
@@ -224,26 +240,19 @@ private BillableUsage produceMonthlyBillable(BillableUsage usage) {
     return usage;
   }
 
-  private boolean updateRemittance(
-      BillableUsageRemittanceEntity remittance, String orgId, BillableUsageCalculation usageCalc) {
-    boolean updated = false;
-    if (!Objects.equals(remittance.getKey().getOrgId(), orgId) && StringUtils.hasText(orgId)) {
-      remittance.getKey().setOrgId(orgId);
-      updated = true;
-    }
-    if (!Objects.equals(remittance.getRemittedValue(), usageCalc.getRemittedValue())) {
-      remittance.setRemittedValue(usageCalc.getRemittedValue());
-      updated = true;
-    }
-    if (!Objects.equals(remittance.getBillingFactor(), usageCalc.getBillingFactor())) {
-      remittance.setBillingFactor(usageCalc.getBillingFactor());
-      updated = true;
-    }
-    // Only update the date if the remittance was updated.
-    if (updated) {
-      remittance.setRemittanceDate(usageCalc.getRemittanceDate());
-    }
-    return updated;
+  private void updateRemittance(BillableUsage usage, BillableUsageCalculation usageCalc) {
+    var newRemittance =
+        BillableUsageRemittanceEntity.builder()
+            .key(BillableUsageRemittanceEntityPK.keyFrom(usage))
+            .build();
+    // Remitted value should be set to usages metric_value rather than billing_value
+    newRemittance.setRemittedPendingValue(
+        usageCalc.getBillableValue() / usageCalc.getBillingFactor());
```

Comment:
This conversion should be replaced with an equivalent value via the Quantity classes we have introduced. That being said, I'm curious as to why this value doesn't come from the usageCalc like it used to. This would avoid having to put another Quantity like conversion here.